README.md

```diff
@@ -88,4 +88,24 @@ resource "spinnaker_pipeline" "terraform_example" {
 
 * `application` - Application name
 * `name` - Pipeline name
-* `pipeline` - Pipeline JSON in string format, example `file(pipelines/example.json)`
\ No newline at end of file
+* `pipeline` - Pipeline JSON in string format, example `file(pipelines/example.json)`
+
+### `spinnaker_project`
+
+#### Example Usage
+
+```
+resource "spinnaker_project" "my_project" {
+    project = "projecttest"
+    email = "howdy@email.com"
+    applications = file("projects/my_project/applications.json")
+    pipelines = file("projects/my_project/pipelines.json")
```

Comment:
i'd like to see these definable as objects instead of reading in the file. it'd be really cool to be able to do...
```
resource "spinnaker_project" "my_project" {
  project = "project"
  email = "howdy@email.com"
  applications = [ "app1", "app2"]
  pipelines = [{application = "app1", pipelineId = "somePipelineId"}]
  clusters = [
    {account = "my-aws-account", detail = "*", stack = "*"}
  ]
}
```

Reply:
Ah, that does look more obvious rather than creating a file-based workflow leading to unnecessary cognitive load when updating/mapping projects. I'll see what I can do here!